be/src/exprs/vectorized/olap_runtime_ranger.h

```diff
@@ -0,0 +1,64 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
```

Comment:
Current, our all files are vectorized. the new files needn't put into `vectorized` directory.